src/Replacers/CsrfTokenReplacer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Spatie\ResponseCache\Replacers;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class CsrfTokenReplacer implements Replacer
+{
+    public function replacedBy(): string
+    {
+        return '<csrf-token-here>';
+    }
+
+    public function transformInitialResponse(Response $response): void
+    {
+        if ($response->getContent()) {
```

Comment:
Let's negate this `if` and put a return statement in it. This way the actual code can be unnested.